bankaccounts.rb

```diff
@@ -6,24 +6,23 @@ class Account
     attr_reader :id, :balance, :open_date
     attr_accessor :owner
 
-    def initialize(id, initial_balance, open_date, owner = nil)
+    def initialize(id, initial_balance, open_date)
       @balance = initial_balance.to_i
       if initial_balance.to_i < 0
           raise ArgumentError, "Invalid Balance: Balance may not be negative."
       end
       @id = id.to_i
       @open_date = DateTime.strptime(open_date, "%Y-%m-%d %H:%M:%S %z")
-      @owner = owner
+      @owner = nil
     end
 
     def withdraw(withdraw_amount)
       if withdraw_amount > @balance
         puts "Warning: Can not withdraw more than is in account. Transaction terminated."
-        return @balance
       else
         @balance -= withdraw_amount
-        return @balance
       end
+    return @balance
```

Comment:
Nice job moving the return to outside of the `if` statement since it applies to both conditions